### PR TITLE
add convtasnet recipe for dns_ins20

### PR DESCRIPTION
**Pull request by muqiaoy**

### CODE REVIEW

File: egs2/dns_ins20/enh1/README.md [hunk 1]
```diff
@@ -44,3 +44,27 @@
...
+  - Commit date: `Sun Feb 6 15:37:51 2022 -0500`
+
+
+## enh_train_enh_conv_tasnet_raw
+
+- config: ./conf/tuning/train_enh_conv_tasnet.yaml
+- model: https://huggingface.co/muqiaoy/muqiaoy_dns_ins20_enh_train_enh_conv_tasnet_raw
+
+
+| dataset                           | STOI | SAR   | SDR   | SIR  |
```
Comment (Emrys365) [inline egs2/dns_ins20/enh1/README.md:66]: Could you remove the SIR column and add SI_SNR?

**Review by Emrys365 — APPROVED** [review]

File: egs2/dns_ins20/enh1/README.md (hunk 1 above)
Comment (muqiaoy) [reply egs2/dns_ins20/enh1/README.md]: Thanks for reminding. I have added the SI_SNR. Could you take a further look?

**Review by Emrys365 — APPROVED** [review]
LGTM.

**Comment by sw005320** [comment]
Thanks, @muqiaoy!

**Pull request merged**